### PR TITLE
fix: leave only minimum version of pandas in Pipfile

### PR DESCRIPTION
**Pull request by joaoandre-avaiga**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ markdown = "==3.4.4"
 marshmallow = "==3.20.1"
 networkx = "==2.6"
 openpyxl = "==3.1.2"
-pandas = ">=1.3.5,<2.3.0"
+pandas = ">=1.3.5"
```
Comment (florian-vuillemot) [inline Pipfile]: Let's always use this version of pandas. The Pipfile.max test will be tried above.
```suggestion
pandas = "==1.3.5"
```

File: taipy/core/Pipfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-pandas = ">=1.3.5,<2.3.0"
+pandas = ">=1.3.5"
```
Comment (florian-vuillemot) [inline taipy/core/Pipfile]:
```suggestion
pandas = "==1.3.5"
```

File: taipy/gui/Pipfile [hunk 3]
```diff
@@ -12,7 +12,7 @@ gevent = "==23.7.0"
 gevent-websocket = "==0.10.1"
 kthread = "==0.2.3"
 markdown = "==3.4.4"
-pandas = ">=1.3.5,<2.3.0"
+pandas = ">=1.3.5"
```
Comment (florian-vuillemot) [inline taipy/gui/Pipfile]:
```suggestion
pandas = "==1.3.5"
```

**Review by jrobinAV — COMMENTED** [review]
I am good with Florian's suggestions.

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**